packages/Babylonian-UI.package/CodeHolder.extension/instance/compiledMethod.st

```diff
@@ -1,4 +1,6 @@
 *Babylonian-UI-private
 compiledMethod
+	 
 	"Only callable on object responding to methodReference, currently BPBrowser and MessageSet"
-	^ self methodReference compiledMethod
\ No newline at end of file
+	^ (self respondsTo: #methodReference)
+		ifTrue: [self methodReference compiledMethod]
```

Comment:
How come you didn't add a case for false? As it stands it would break when it doesn't respond to #methodReference  which would not change the behavior from the previous version

Reply:
I think I do not yet understand your concern 😅 If the receiver does not respond to `#methodReference`, `#compiledMethod` will answer `nil`, and so will do `#exampleForEvaluation`. When exactly would this break?

Reply:
Ah the current version returns `nil` if `self respondsTo: #methodReference` is false? I thought it would return self and might break as self also is not nil.

Reply:
> Ah the current version returns `nil` if `self respondsTo: #methodReference` is false?

Yes

> I thought it would return self

Nope ... see `False >> #ifTrue:` :P

Reply:
Ok, something new learned then :)